java/rocksjni/options_util.cc

```diff
@@ -51,40 +51,12 @@ void build_column_family_descriptor_list(
   }
 }
 
-/*
- * Class:     org_rocksdb_OptionsUtil
- * Method:    loadLatestOptions
- * Signature: (Ljava/lang/String;JLjava/util/List;Z)V
- */
-void Java_org_rocksdb_OptionsUtil_loadLatestOptions__Ljava_lang_String_2JJLjava_util_List_2Z(
-    JNIEnv* env, jclass /*jcls*/, jstring jdbpath, jlong jenv_handle,
-    jlong jdb_opts_handle, jobject jcfds, jboolean ignore_unknown_options) {
-  jboolean has_exception = JNI_FALSE;
-  auto db_path =
-      ROCKSDB_NAMESPACE::JniUtil::copyStdString(env, jdbpath, &has_exception);
-  if (has_exception == JNI_TRUE) {
-    // exception occurred
-    return;
-  }
-  std::vector<ROCKSDB_NAMESPACE::ColumnFamilyDescriptor> cf_descs;
-  ROCKSDB_NAMESPACE::Status s = ROCKSDB_NAMESPACE::LoadLatestOptions(
-      db_path, reinterpret_cast<ROCKSDB_NAMESPACE::Env*>(jenv_handle),
-      reinterpret_cast<ROCKSDB_NAMESPACE::DBOptions*>(jdb_opts_handle),
-      &cf_descs, ignore_unknown_options);
-  if (!s.ok()) {
-    // error, raise an exception
-    ROCKSDB_NAMESPACE::RocksDBExceptionJni::ThrowNew(env, s);
-  } else {
-    build_column_family_descriptor_list(env, jcfds, cf_descs);
-  }
-}
-
 /*
  * Class:     org_rocksdb_OptionsUtil
  * Method:    loadLatestOptions_1
```

Comment:
I suppose we can remove the `_1` suffix now

Reply:
Good catch. Thank you for your thorough review!